server/server/src/billing/app_store_client.rs

```diff
@@ -0,0 +1,92 @@
+use crate::billing::app_store_model::{VerifyReceiptRequest, VerifyReceiptResponse};
+use crate::config::AppleConfig;
+use jsonwebtoken::{encode, Algorithm, EncodingKey, Header};
+use lockbook_shared::clock::get_time;
+use reqwest::RequestBuilder;
+use serde::{Deserialize, Serialize};
+
+const VERIFY_PROD: &str = "https://buy.itunes.apple.com/verifyReceipt";
+const VERIFY_SANDBOX: &str = "https://sandbox.itunes.apple.com/verifyReceipt";
+
+const ALG: &str = "ES256";
```

Comment:
can this change?

Reply:
It cannot, it is a part of [apple's standards](https://developer.apple.com/documentation/appstoreserverapi/generating_tokens_for_api_requests).